test/TestHelpers.cs

```diff
@@ -182,6 +188,20 @@ private static List<string> GetLogs_HelloWorldOrchestration_Inline(string messag
             return list;
         }
 
+        private static List<string> GetLogs_OrchestrationEventGridApiReturnBadStatus(string messageId, string[] functionNames)
+        {
+            var list = new List<string>()
+            {
+                $"{messageId}: Function '{functionNames[0]} ({FunctionType.Orchestrator})', version '' scheduled. Reason: NewInstance. IsReplay: False. State: Scheduled. HubName: OrchestrationStartAndCompleted. AppName: . SlotName: . ExtensionVersion: 1.2.1.0. SequenceNumber: 0.",
```

Comment:
Let's not check for HubName, AppName, SlotName, ExtensionVersion, or SequenceNumber. Checking for EventVersion is especially problematic because this test will break every time we release a new update.  See the other GetLogs_ implementations for an example of what is okay to check for.

Reply:
Done!